cmd/zc_traverser_blob.go

```diff
@@ -180,7 +215,7 @@ func (t *blobTraverser) parallelList(containerURL azblob.ContainerURL, container
 		currentDirPath := dir.(string)
 		for marker := (azblob.Marker{}); marker.NotDone(); {
 			lResp, err := containerURL.ListBlobsHierarchySegment(t.ctx, marker, "/", azblob.ListBlobsSegmentOptions{Prefix: currentDirPath,
-				Details: azblob.BlobListingDetails{Metadata: true}})
+				Details: azblob.BlobListingDetails{Metadata: true, Tags: true}})
```

Comment:
>Tags: true [](start = 55, length = 10)

Including tags may incur a perf hit, maybe we should only do it if we need the tag info.